src/org/parosproxy/paros/view/MainMenuBar.java

```diff
@@ -114,10 +128,57 @@ private void initialize() {
 			menuEdit = new javax.swing.JMenu();
 			menuEdit.setText(Constant.messages.getString("menu.edit")); // ZAP: i18n
 			menuEdit.setMnemonic(Constant.messages.getChar("menu.edit.mnemonic"));
+			menuEdit.add(getMenuEditZAPMode());
 		}
 		return menuEdit;
 	}
 
+	private JMenuItem getMenuEditZAPMode(){
+		if (menuZAPMode == null) {
+			menuZAPMode = new JMenu(Constant.messages.getString("menu.edit.zapmode"));
+			menuZAPModeGroup = new ButtonGroup();
+			JRadioButtonMenuItem newButton = null;
+			for (Mode modeType : Mode.values()) {
+				 newButton = addZAPModeMenuItem(modeType);

```

Comment:
In this and following two lines there are some spaces at the beginning of the statements and between the tabs.